apis/kueue/v1beta1/clusterqueue_types.go

```diff
@@ -91,6 +91,7 @@ type ClusterQueueSpec struct {
 
 	// admissionChecks lists the AdmissionChecks required by this ClusterQueue
 	// +optional
+	// +kubebuilder:validation:XValidation:rule="!('kueue-preemption' in self)",message="Admission name 'kueue-preemption' is reserved for internal kueue use"
```

Comment:
Let's not use CEL yet. Write it in the webhook.

Reply:
done